xpdacq/Ni24.D

```diff
@@ -0,0 +1,27 @@
+# Calibrant: Nickel (Ni)
```

Comment:
Haha it's just a minor edit to our trusty Ni03.cal d-spacings from fit2d, but has more spacings than the Ni.D deafault in the PyFAI library. This just helps with refining the detector non-orthogonality.